rollup.config.mjs

```diff
@@ -2,14 +2,27 @@ import { nodeResolve } from '@rollup/plugin-node-resolve';
 import typescript from '@rollup/plugin-typescript';
 import postcss from 'rollup-plugin-postcss';
 
+/**
+ * @type {import('rollup').RollupOptions}
+ */
 export default {
   input: 'src/index.ts',
-  output: {
-    dir: 'lib',
-    format: 'es',
-    preserveModules: true,
-    preserveModulesRoot: 'src',
-  },
+  output: [
+    {
+      dir: 'lib/es',
+      format: 'es',
+      preserveModules: true,
+      preserveModulesRoot: 'src',
+      sourcemap: true,
+    },
+    {
+      dir: 'lib/cjs',
+      format: 'cjs',
+      preserveModules: true,
+      preserveModulesRoot: 'src',
+      sourcemap: true,
```

Comment:
We can turn sourcemaps off, not sure if anyone will take a look but also wouldn't hurt to have